osu.Game/Configuration/TaikoTouchControlType.cs

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+#nullable disable
+
+namespace osu.Game.Configuration
+{
+    public enum TaikoTouchControlType
+    {
+        Default, // KDDK
+        DDKK,
+        KKDD
```

Comment:
Probably best to use more descriptive names for the control type rather than community terms, e.g. Katsu-don-don-katsu or any other descriptive name.